src/utils/update-config.js

```diff
@@ -29,7 +29,13 @@ const parseXML = function(xmlPath) {
 };
 
 const saveXML = function(json, xmlPath) {
-  const builder = new xml2js.Builder();
+  const builder = new xml2js.Builder({
+    renderOpts: {
+      'pretty': true,
+      'indent': '    ',
+      'newline': '\n'
```

Comment:
For now we're gonna leave this as a `\n` as we have other more pressing items. 

That said, as @alexblom [said](https://github.com/isleofcode/splicon/issues/12#issuecomment-253307396) we're moving towards treating `config.xml` as a temporary file. And with that in mind, once it is committed as a file with `\n` line endings, subsequent changes should (also using the `\n` line endings) should just show the single lines changed.